src/pages/Graph/GraphPage.tsx

```diff
@@ -199,6 +201,10 @@ export class GraphPage extends React.Component<GraphPageProps, GraphPageState> {
   private graphDataSource: GraphDataSource;
 
   static getNodeParamsFromProps(props: RouteComponentProps<Partial<GraphURLPathProps>>): NodeParamsType | undefined {
+    const aggregate = props.match.params.aggregate;
+    const aggregateOk = aggregate && aggregate !== UNKNOWN && aggregate !== 'undefined';
```

Comment:
I see `aggregate !== 'undefined'`. Is it true that `aggregate` can be the `"undefined"` string? or are you testing if it's an undefined variable?

If it's the latter, I think the very first test is already checking for `undefined` variable (not string).

Reply:
good point, should not be testing a string.

Reply:
@israel-hdez I removed the invalid tests here and in a few other places where the same problem had been originally.